README.md

```diff
@@ -71,10 +71,13 @@ running
 You may need to run
 
     sysctl -w kernel.perf_event_paranoid=-1
+
+Additionally, if you want to use sandboxing on older kernels, you'll need to run
```

Comment:
If we're to be ultra-hyper-super-precise, @Wolf480pl says that this only applies to older kernels on Debian (because it was a Debian-specific option all along).